client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -541,91 +526,91 @@ def download(self, dest_path: Optional[str] = None):
+        https_collection_metadata (str): None
+        https_mrc_bin1 (str): None
+        https_omezarr_dir (str): None
+        id (int): None
+        is_aligned (bool): Tilt series is aligned
+        microscope_additional_info (str): Other microscope optical setup information, in addition to energy filter, phase plate and image corrector
+        microscope_energy_filter (str): Energy filter setup used
         microscope_image_corrector (str): Image corrector setup
         microscope_manufacturer (str): Name of the microscope manufacturer
         microscope_model (str): Microscope model name
         microscope_phase_plate (str): Phase plate configuration
-        pixel_spacing (float): Pixel spacing for the tilt series
-        related_empiar_entry (str):  If a tilt series is deposited into EMPIAR, enter the EMPIAR dataset identifier
-        s3_alignment_file (str): S3 path to the alignment file for this tiltseries
-        s3_angle_list (str): S3 path to the angle list file for this tiltseries
-        s3_collection_metadata (str): S3 path to the collection metadata file for this tiltseries
-        s3_mrc_bin1 (str): S3 path to this tiltseries in MRC format (no scaling)
-        s3_omezarr_dir (str): S3 path to this tomogram in multiscale OME-Zarr format
+        pixel_spacing (float): Pixel spacing equal in both axes in angstrom
+        related_empiar_entry (str): If a tilt series is deposited into EMPIAR, enter the EMPIAR dataset identifier
+        run (Run): An object relationship
+        run_id (int): None
+        s3_alignment_file (str): None
+        s3_angle_list (str): None
+        s3_collection_metadata (str): None
+        s3_mrc_bin1 (str): None
+        s3_omezarr_dir (str): None
         spherical_aberration_constant (float): Spherical Aberration Constant of the objective lens in millimeters
         tilt_axis (float): Rotation angle in degrees
         tilt_max (float): Maximal tilt angle in degrees
         tilt_min (float): Minimal tilt angle in degrees
-        tilt_range (float): Total tilt range in degrees
+        tilt_range (float): The difference between tilt_min and tilt_max
         tilt_series_quality (int): Author assessment of tilt series quality within the dataset (1-5, 5 is best)
-        tilt_step (float): Tilt step in degrees
+        tilt_step (float): None
         tilting_scheme (str): The order of stage tilting during acquisition of the data
         total_flux (float): Number of Electrons reaching the specimen in a square Angstrom area for the entire tilt series
     """
 
-    _gql_type = "tiltseries"
-
-    run: Run = ItemRelationship(Run, "run_id", "id")
+    _gql_type: str = "tiltseries"
 
     acceleration_voltage: int = IntField()
     aligned_tiltseries_binning: int = IntField()
     binning_from_frames: float = FloatField()
     camera_manufacturer: str = StringField()
     camera_model: str = StringField()
-    data_acquisition_software: float = FloatField()
```

Comment:
Looks like a copy/paste error. This works on `main` because the definiton of `FloatField` and `StringField` currently identical (i.e. they just extend `BaseField` without overriding or adding anything).